src/inc/clrconfigvalues.h

```diff
@@ -529,14 +529,18 @@ CONFIG_DWORD_INFO_EX(INTERNAL_JitLoopHoistStats, W("JitLoopHoistStats"), 0, "Dis
 CONFIG_DWORD_INFO_EX(INTERNAL_JitDebugLogLoopCloning, W("JitDebugLogLoopCloning"), 0, "In debug builds log places where loop cloning optimizations are performed on the fast path.", CLRConfig::REGUTIL_default);
 CONFIG_DWORD_INFO_EX(INTERNAL_JitVNMapSelLimit, W("JitVNMapSelLimit"), 0, "If non-zero, assert if # of VNF_MapSelect applications considered reaches this", CLRConfig::REGUTIL_default)
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_JitVNMapSelBudget, W("JitVNMapSelBudget"), 100, "Max # of MapSelect's considered for a particular top-level invocation.")
-#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
+#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_) || defined(_TARGET_ARM64_)
 #define EXTERNAL_FeatureSIMD_Default 1
-#define EXTERNAL_JitEnableAVX_Default 1
-#else // !defined(_TARGET_AMD64_) && !defined(_TARGET_X86_)
+#else // !(defined(_TARGET_AMD64_) || defined(_TARGET_X86_) || defined(_TARGET_ARM64_))
 #define EXTERNAL_FeatureSIMD_Default 0
+#endif // !(defined(_TARGET_AMD64_) || defined(_TARGET_X86_) || defined(_TARGET_ARM64_))
+#if defined(_TARGET_XARCH_)
```

Comment:
@BruceForstall Does this need to be `#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)` ?

Reply:
yes.

Reply:
I'm confused how does:
```
#if defined(_TARGET_AMD64_) || defined(_TARGET_X86_)
```
differ from what's there:
```
#if defined(_TARGET_XARCH_)
```

Reply:
In previous changes @BruceForstall has indicated that `_TARGET_XARCH_` should not be used in certain files.

Reply:
Ah I see - thanks for the clarification.

Reply:
In particular, it's only defined for the JIT, and clrconfigvalues.h is included by the VM.

Reply:
Done